docs/content/docs/test/ScreenChecks.md

```diff
@@ -0,0 +1,96 @@
+# Screen checks
+
+`Screen` is abstraction that represents `Activity`, `Fragment` or `View` in tests. \
+`Screen` has a property `checks` of type `ScreenChecks` with single function `isScreenOpened` \ 
+`ScreenChecks` has a property `checkOnEachScreenInteraction` which is responsible for: If we shall execute `isScreenOpened` on each interaction with `ViewElement` on that `Screen` \
+
+Default behavior is:
+- `Screen.checks` type is `StrictScreenChecks`
+- `checkOnEachScreenInteraction` value is `false`
+- `isScreenOpened` checks if `rootView.id` equals to `Screen.rootId` 
+
+## How to use ScreenChecks directly?
+
+```kotlin
+import com.avito.android.screen.Screen
+
+val screen: Screen = TODO("Logic for initializing your Screen")
+screen.checks.isScreenOpened()
+```
+
+## How to customize `isScreenOpened` check
```

Comment:
I'd be good to add a short explanation about reasons to do so.

Reply:
Done